### ISSUE TITLE
AngelScript is not worked

### ISSUE THREAD

**Issue by zulu093** [RigsOfRods/rigs-of-rods#1367]
Hello.

I want to play the new developed MAP and AS using AI. However, error message(Failed to build the module) is displayed and AS is not executed.

I think the part containing parseInt () seems to be missing at compile time. because the remaining part operates normally.

What I mean is that parserint () does not work when called.

I am currently using AngelScript 2.21.1, Ubuntu 17.04 and RoR 0.4.7.0-Dev.

I want to know how to solve this problem.

If someone knows the solution to the problem, please reply.

===================================================================
[Rigs of Rods log]

```
Unkown error while loading script file: //If your map has a race script define it here in this format
Failed to add script file
races.as (251, 2): Info = Compiling void racesManager::raceEvent(int, string, const string&in, int)
races.as (261, 25): Error = No matching signatures to 'parseInt(string)'
races.as (262, 25): Error = No matching signatures to 'parseInt(string)'
races.as (295, 25): Error = No matching signatures to 'parseInt(string)'
races.as (296, 25): Error = No matching signatures to 'parseInt(string)'
races.as (354, 24): Error = No matching signatures to 'parseInt(string)'
races.as (355, 24): Error = No matching signatures to 'parseInt(string)'
races.as (1016, 2): Info = Compiling void racesManager:ConfusedetupArrow(int)
races.as (1030, 24): Error = No matching signatures to 'parseInt(string)'
races.as (1032, 19): Error = No matching signatures to 'parseInt(string)'
races.as (1440, 2): Info = Compiling void raceBuilder:ConfusedaveRace(LocalStorage@)
races.as (1445, 26): Error = No matching signatures to 'string::findFirst(string, const uint)'
races.as (1446, 26): Error = No matching signatures to 'string::findFirst(string, const uint)'
races.as (1469, 2): Info = Compiling void raceBuilder::loadRace(LocalStorage@)
races.as (1474, 26): Error = No matching signatures to 'string::findFirst(string, const uint)'
races.as (1475, 26): Error = No matching signatures to 'string::findFirst(string, const uint)'
races.as (1498, 13): Error = No matching signatures to 'string::findFirst(string, int)'
races.as (1499, 27): Error = No matching signatures to 'parseFloat(string)'
Failed to build the module
saving script bytecode to file 
```
![ubuntu 64-bit-2017-08-16-18-52-05](https://user-images.githubusercontent.com/20366627/29357882-23c80a1c-82b4-11e7-8f53-8d18c385b26c.png)

**Comment by only-a-ptr** [comment]
Hello. 

I've also been getting strange AngelScript failures lately. I'm worried the problem is not related to AngelScript, but inherent to current upstream codebase: https://github.com/RigsOfRods/rigs-of-rods/issues/1366

**Comment by zulu093** [comment]
First, thank you for your answer.

I want to use races.as, how can I fix the problem?

**Comment by only-a-ptr** [comment]
I don't know what's a "laptime function", sorry. But I suggest you try to find a solution rather than a workaround.

Are you building RoR from upstream or using a release? Try building from older revisions until you find where it broke for you.

**Comment by zulu093** [comment]
I am working on a project to develop simulators similar to real cars using the RoR engine.

I needed to measure Laptime for the simulator function, so I tried to use AngelScript.

In my windows environment, my Terrain and Script worked fine.

However, it did not work on Linux (Ubuntu 17.04) environment 

What I tried was installing older versions of AngelScript, installing older versions of RoR, and installing older versions of Ubuntu.

But all failed, eventually abandoning AngelScript. Now I am developing a Laptime function using C ++ And personally, RoR seems to be a really cool car simulator. Thank you for seeing the RoR project.

I am really grateful to the people who have created this wonderful project.
